.gitpod.yml

```diff
@@ -34,6 +34,15 @@ tasks:
   - name: Go
     init: leeway exec --filter-type go -v -- go get -v ./...
     openMode: split-right
+  - name: Docker
+    command: |
+      if [ -x "$(which docker-up)" ] 
```

Comment:
I had a bit more of a thought about this. While it's true that `docker-up` is only in the path if feature preview is enabled, that's because feature preview also enables `registry-facade`. Hence `which docker-up` is more of a signal for registry facade rather than user namespaces.
Also, in gitpod-io it's likely to have `docker-up` in the path even without registry-facade due to the `go get ...` in `components/docker-up`.

Instead you could check if `sudo` works, e.g.
```bash
if [ sudo true &>/dev/null ]; then
  sudo docker-up
else
  echo "Enable 'Feature Preview' in your Gitpod profile for Docker support."
fi
```